examples/data-objects/prosemirror/src/fluidBridge.ts

```diff
@@ -142,12 +142,13 @@ export class ProseMirrorTransactionBuilder {
                 //
                 // For positions we *will* need to include any newly inserted nodes. We can count these as "new" ether
 
-                assert(i < this.things.length);
+                assert(i < this.things.length, "Trying to insert removal node out-of-bounds!");
 
                 i = this.splitAt(position, i);
                 let length = range.segment.cachedLength;
                 while (length > 0) {
-                    assert(this.things[i].type === "ether");
+                    assert(this.things[i].type === "ether",
+                        `Current thing does not have 'ether' node type! Has '${this.things[i].type}' type.`);
```

Comment:
don't put varialbes in assert messages. it makes them really hard to query. if we need variable, we should log or throw an error with the varialbe as a property